pages/linux/trap.md

```diff
@@ -0,0 +1,24 @@
+# trap
+
+> Automatically execute commands after receiving signals by processes or the operating system.
+> Can be used to perform cleanups for interruptions by the user or other causes.
+
+- List available signals to set traps for:
+
+`trap -l`
+
+- List active traps for the current shell:
+
+`trap -p`
+
+- Set a trap to execute commands when one or more signals are detected:
+
+`trap '{{shell_commands}}' {{signal_names}}`
```

Comment:
You can mention the example here itself. By convention, we prefer to have concrete examples rather than generic ones. And perhaps a better command like `echo "Caught signal SIGHUP"` to match the signal type.